internal/support/scheduler/schedule.go

```diff
@@ -9,15 +9,16 @@ package scheduler
 import (
 	"errors"
 	"fmt"
-	"github.com/edgexfoundry/edgex-go/pkg/models"
-	queueV1 "gopkg.in/eapache/queue.v1"
-	"gopkg.in/mgo.v2/bson"
 	"io/ioutil"
 	"net/http"
 	"regexp"
 	"strings"
 	"sync"
 	"time"
+
+	"github.com/edgexfoundry/edgex-go/pkg/models"
+	queueV1 "gopkg.in/eapache/queue.v1"
+	"gopkg.in/mgo.v2/bson"
```

Comment:
Why do we do this anyway -- putting all of the third party stuff at the bottom? By default the IDE (GoLand) alphabetizes imports when they're added. Then we have to manually move this stuff around. Any downside to leaving them alphabetized?

Reply:
I work using atom with [go-plus](https://atom.io/packages/go-plus). Among other plugins, go-plus includes [goimports](https://godoc.org/golang.org/x/tools/cmd/goimports), which automatically sorts the imports in the way you have seen every time I save a file, putting all third party stuff at the bottom.